app/views/plans/_share_form.html.erb

```diff
@@ -128,7 +128,7 @@
       <h2><%= _('Request expert feedback') %></h2>
       <p><%= _('Click below to give data management staff at your organisation access to read and comment on your plan.') %></p>
       <div class="well well-sm">
-        <%= sanitize current_user.org.feedback_email_msg.to_s %>
+        <%= sanitize current_user.org.feedback_email_msg.to_s.gsub("%{user_name}", current_user.name(false)).gsub("%{plan_name}", plan.title) %>
```

Comment:
```suggestion
        <%= sanitize current_user.org.feedback_email_msg.to_s % {user_name: current_user.name(false), plan_name: plan.title} %>
```

I think that you could replace the two gsub calls with the above

Reply:
@xsrust @briri  Updated with change suggested by @xsrust and forced pushed.

Reply:
suggested changes were made